.github/workflows/deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: "Deploy to PyPI"
+on:
+  workflow_dispatch:
+    inputs: 
+      tag:
+        description: "Git tag to deploy to PyPI"
+        required: true
+
+jobs:
+  deploy:
+    name: "Deploy to PyPI"
+    runs-on: ubuntu-latest
+    environment: Deploy
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
Picky hat still on : `actions/checkout@v2` is available.

Reply:
Sure